libnetwork/iptables/firewalld.go

```diff
@@ -70,6 +71,12 @@ var (
 func FirewalldInit() error {
 	var err error
 
+	// When running with RootlessKit, firewalld is running as the root outside our namespace, so we do not have the permission
```

Comment:
This is less about not having permission (firewalld uses polkit which asks the user interactively, if there's someone physically at the console to ask) and more about firewalld configuring rules in the wrong network namespace.

Reply:
fixed